src/sql/platform/dialog/media/wizardNavigation.css

```diff
@@ -7,8 +7,7 @@
 	display: flex;
 	flex-direction: column;
 	width: 80px;
-	height: calc(100% + 25px);
-	margin-top: -25px;
```

Comment:
Great to get rid of the `calc` heights and hacky margins here and in the other css files above. Thanks!